launch_ros/launch_ros/actions/node.py

```diff
@@ -14,8 +14,10 @@
 
 """Module for the Node action."""
 
+import importlib_metadata
 import os
 import pathlib
+from packaging.version import Version
```

Comment:
Third party package, should be in the same block as `yaml`

Reply:
Fixed in https://github.com/ros2/launch_ros/pull/216/commits/ded9e1d9f33cef40762a43310af29e9cb374cebe